### ISSUE TITLE
Timed pictures may be lost in lymbo

### ISSUE THREAD

**Issue by IpsmLorem** [wireapp/wire-ios#435]
Something wierd happend (trying to figure out the sequence):

* User A send 6 timed pictures (5') to user B
* User A send several regular messages and pictures
* User B open Wire on Android and see only the regular messages
* User B wire may have triggered the count down fuse
* User B scroll up after time up and cannot see the pictures
* User A have still the shadow timed pictures on their conversation display, like if they were not read and cannot remove them

**Comment by KoalaBoomka** [comment]
hi @IpsmLorem !
What did you do in step 3 for seeing only regular messages?
If user B on Android got several timed and regular messages, then he will see the feed from the first unread message. It means that he will scroll through all the messages and trigger a timer for timed messages.

**Comment by IpsmLorem** [comment]
Yes it triggers but nor read confirmation is sent (received ?) to A and then timeds medias shadow is forever displayed on the A device (since a timed sending (while valid or shadowed) cannot be removed.

**Comment by IpsmLorem** [comment]
Happend with two android devices as target.

**Comment by jenskasemets** [comment]
@hakonbo could you please check if it might be on Android side?

**Comment by IpsmLorem** [comment]
Still happening. I noticed that I'm not receiving delivered confirmation nor timed message trigger from some of my Android contacts.

**Comment by IpsmLorem** [comment]
Bug still present and is very annoying. It ends up flooding my screen with unconfirmed timed messages.
